src/frontend/src/lib/services/address.services.ts

```diff
@@ -89,6 +98,11 @@ const bitcoinMapper: Record<
 	testnet: {
 		addressStore: btcAddressTestnetStore,
 		setIdbAddress: setIdbBtcAddressTestnet
+	},
+	regtest: {
+		addressStore: btcAddressRegtestStore,
+		// No need to store the regtest in the local storage because it's only used locally.
+		setIdbAddress: undefined
```

Comment:
If it's a choice, `setIdbAddress` shouldn't be set to `null` instead of `undefined`?